card_kind.py

```diff
@@ -0,0 +1,6 @@
+from enum import Enum
+
+
+class Kind(Enum):
+    ANSWER = "answer"
+    QUESTION = "question"
```

Comment:
Do we need the values of the enum or can we set them to default?

Reply:
You might want to change it into a regular class and add this parameter as `type_name`